### DISCUSSION TITLE
4.9 install on Windows 10  throws org.osgi.framework.BundleException

### DISCUSSION THREAD

**Discussion by GingerJohnny** [eclipse-birt/birt#1012] [General]
Can someone provide some insight?

I just unpacked birt-report-designer-all-in-one-4.9.0-20220315-win32.win32.x86_64.zip on my Windows 10 machine. When I start either the birt.exe or the eclipse.exe, an error log is created with the following:

!SESSION 2022-07-12 10:19:28.856 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_333
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.equinox.simpleconfigurator 4 0 2022-07-12 10:19:29.412
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException:  [3]
  Unresolved requirement: Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"

**Comment by claesrosell** [comment]
Hi.

You need JavaSE-11 or higher to run Birt 4.9.
